.travis.yml

```diff
@@ -2,6 +2,17 @@ language: python
 sudo: false
 python:
   - 2.7
+  - 3.5
+  - 3.6
+  - 3.7
```

Comment:
Python 3.7 does not run on Debian Trusty, so it should be removed here. (It is run on Debian Xenial via the `matrix`.